cmd/metal-api/internal/datastore/rethinkdb_test.go

```diff
@@ -85,31 +85,6 @@ func TestRethinkStore_db(t *testing.T) {
 	}
 }
 
-func TestRethinkStore_Mock(t *testing.T) {
```

Comment:
This test is rather useless and also the linter does not like it, therefore removed.